Basic/APIDiorama/Assets/Common/Scripts/DioramaUtilities.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using UnityEngine;
+
+namespace Unity.Netcode.Samples.APIDiorama
+{
+    /// <summary>
+    /// A class of utilities used across Dioramas
+    /// </summary>
+    public static class DioramaUtilities
+    {
+        static readonly string[] s_Usernames = new string[] { "MorwennaDaBest", "SamTheBell", "FranklyJil", "FernandoCutter", "LP Morgan" };
+
+        /// <summary>
+        /// Generates a random color
+        /// </summary>
+        /// <returns>A random RGBA color</returns>
+        public static Color32 GetRandomColor() => new Color32((byte)UnityEngine.Random.Range(0, 256), (byte)UnityEngine.Random.Range(0, 256), (byte)UnityEngine.Random.Range(0, 256), 255);
+        public static string GetRandomUsername() => s_Usernames[UnityEngine.Random.Range(0, s_Usernames.Length)];
+
+        public static string FilterBadWords(string input)
```

Comment:
What Netcode-related educational content would filtering provide for users? I'm just thinking that we should be as light as possible on code that we'd need to maintain later down the line, and this is something that another service could provide. We should just be teaching the barebones usage of API, right?

I'm not very comfortable with explicitly having these words written out in our sample; it will offend some users. I also don't feel comfortable having my name used in a sample. Can we use generic names for players instead?

Reply:
> I also don't feel comfortable having my name used in a sample. Can we use generic names for players instead?

Sure, I'll change them in the next commit.

> I'm not very comfortable with explicitly having these words written out in our sample; it will offend some users.

I get it. We can show the same feature with more innocent words.

> What Netcode-related educational content would [chat] filtering provide for users? 

It's one of the features that should be done with the APIs we're explaining to users in that scene (RPCs + server-side checks / computation). 
Can you think of another commonly used feature that meets the same criteria?

> We should just be teaching the barebones usage of API, right?

Not really. What would be the difference between this sample and the code in the online documentation, then?

What users have been asking a lot is a sample that shows how to use the APIs for "real" use cases, and in-game chats fall into this category.

Reply:
> What users have been asking a lot is a sample that shows how to use the APIs for "real" use cases, and in-game chats fall into this category.

Ah I see, that context clears things up. Looks good then! I do like the change to renaming it from ChatManager to MoodManager, good call.